### PR TITLE
add bootstrap and font awesome icons

### LINKED ISSUE #26: add bootstrap and font awesome icons
# REQUIREMENTS
Install Bootstrap and Font Awesome Icons to make a better design by using ruby gems

# DESIGN
No Bootstrap and Font Awesome Icons 
image
With  Bootstrap and Font Awesome Icons 
image

# Acceptance Criteria
- [x] Bootstrap can be used in all views without adding any html link or script tag
- [x] Font Awesome Icons can be used in all views without adding any html link or scrip tag

### PR DESCRIPTION
**Pull request by MartinSGL**
adding bootstrap and font awesome icons by using ruby gems

# Description
 - Add Bootstrap and Font Awesome Icons by using ruby gems
 - Change the application.css to applicaction.scss
 - Import Bootstrap and Font Awesome Icons
 - Require jquery, popper and bootstrap in application.js
 
 Examples:
 - I have added bootstrap and font awesome icons by using gems files
 - These changes make easier to harvest the code necessary for design.
 
 # Testing
add an icon and class of bootstrap to test that it's works
  
  ## Screenshots 
image

### CODE REVIEW

File: reloj-checador/Gemfile [hunk 1]
```diff
@@ -18,6 +18,13 @@ gem 'turbolinks', '~> 5'
 
 gem 'devise'
 
+#bootstrap
+gem 'bootstrap', '~> 4.3.1'
+#jquery3
+gem 'jquery-rails'
```
Comment (juliocabrera820) [inline reloj-checador/Gemfile:24]: We can not use jQuery

File: reloj-checador/app/javascript/packs/application.js [hunk 2]
```diff
@@ -11,3 +11,8 @@ import "channels"
 Rails.start()
 Turbolinks.start()
 ActiveStorage.start()
+
+//= require jquery3
```
Comment (juliocabrera820) [inline reloj-checador/app/javascript/packs/application.js:15]: We can not use jQuery

File: reloj-checador/app/assets/stylesheets/application.scss [hunk 3]
```diff
@@ -13,3 +13,6 @@
  *= require_tree .
  *= require_self
  */
+ 
+ @import "bootstrap";
+ @import 'font_awesome5_webfont';
```
Comment (juliocabrera820) [inline reloj-checador/app/assets/stylesheets/application.scss:18]: You can also use double quotes

**Review by juliocabrera820 — COMMENTED** [review]
jQuery must be removed

**Review by jairolan — APPROVED** [review]

**Review by juliocabrera820 — APPROVED** [review]
LGTM

**Pull request merged**